### PR TITLE
docs for ip allowlist/denylist

### PR DESCRIPTION
**Pull request by zhaohuabing**
close: https://github.com/envoyproxy/gateway/issues/3715

### CODE REVIEW

File: site/content/en/latest/tasks/security/restrict-ip-access.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+---
+title: "IP Whitelisting/Blacklisting"
```
Comment (arkodg) [inline site/content/en/latest/tasks/security/restrict-ip-access.md:2]: lets replace these terms as highlighted by the CNCF INI doc
https://inclusivenaming.org/word-lists/tier-1/whitelist/

File: site/content/en/latest/tasks/security/restrict-ip-access.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+Send a request to the backend service with the `X-Forwarded-For` header:
+
+```shell
+curl -v -H "Host: www.example.com" -H "X-Forwarded-For: 10.0.1.1" "http://${GATEWAY_HOST}/"
+```
+
+The request should be allowed and you should see the response from the backend service.
+
+```shell
```
Comment (Xunzhuo) [inline site/content/en/latest/tasks/security/restrict-ip-access.md:178]: This is not in the correct place

**Comment by zhaohuabing** [comment]
@guydc This docs PR doesn't need to be cherry-picked to v1.1.0.  We need to create a v1.1.0 directory and copy the latest docs there.

**Comment by Xunzhuo** [comment]
@zhaohuabing the v1.1 folder will be generated when releasing v1.1, so we just need to merge this into latest for now.

**Review by guydc — APPROVED** [review]

**Pull request merged**